src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -198,12 +198,18 @@ private async Task StartContainerAsync(IExecutionContext executionContext, Conta
                 }
             }
 
+            // TODO: Add at a later date. This currently no local package registry to test with
+            // UpdateRegistryAuthForGitHubToken(executionContext, container);
```

Comment:
finish todo?

Reply:
This would either be added in the near future, or Id remove the TODO if we decide to not do the feature. Basically I couldnt figure out how to verify the change because I dont know how to run a local/development package registry. Since we (at least I do) test certain changes with a development runner against prod actions, I was going to test and finally add this feature back in once actions service changes were deployed 

A quick search looked like we have some TODOs in the runner code, so I didnt feel like it was breaking any established pattern

Ill open a small PR resolving this particular TODO soon